7-extract_plier_pathways.R

```diff
@@ -17,7 +17,10 @@ option_list <- list(
   optparse::make_option("--ncores",
                         default = NA_integer_,
                         help = "Set the number of cores to use"
-  )
+  ),
+  optparse::make_option("--permute",
+                        default = FALSE,
```

Comment:
Based on the name of this option, I would kind of expect this to be `action = "store_true"`. So to run PLIER with the true gene-pathway relationships, I'd run:

```
Rscript 7-extract_plier_pathways.R --cancer_type $cancer_type --ncores $ncores
```

And to permute the gene-pathway relationships, I'd run:

```
Rscript 7-extract_plier_pathways.R --cancer_type $cancer_type --ncores $ncores --permute
```

I don't have strong feelings that this needs to be changed. I just wanted to give you a bit of feedback re: my expectations based on what you have!